_posts/2022-11-15-cryostat-release.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: Cryostat 2.2.0 is Released!
+date: 2022-11-15
+synopsis: Cryostat 2.2.0 is here. Learn what's new in this release.
+author: Andrew Azores
+
+---
+
+Hello readers, and welcome back to the Cryostat blog. Today we have exciting news: Cryostat 2.2.0 is released!
+
+Upstream container images and assets are available at the usual sources:
+- [quay.io/repository/cryostat/cryostat](https://quay.io/repository/cryostat/cryostat?tab=tags)
```

Comment:
Not sure if this was intended, but maybe keep the links consistent? (`?tab=tags` for cryostat link)